### PR TITLE
Cleanup CG manifest generation

### PR DESCRIPTION
**Pull request by tiagoshibata**
Cleanup Component Governance manifest generation, removing duplicates with same commit.

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -1,461 +1,641 @@
...
-   "Version": 1
+    "Registrations": [
+        {
+            "component": {
+                "git": {
+                    "commitHash": "1b543d6e5073b56be214394890c9193979a3d7e1",
+                    "repositoryUrl": "https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer"
+                },
+                "type": "git"
+            }
```
Comment (skottmckay) [inline cgmanifest.json]: What is the reason for changing the ordering of everything and having such a large diff? 
Did the content change at all besides the re-ordering?

File: tools/python/get_submodules.py [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
 
+def absolute_url(url):
+    # Support relative URLs (see https://github.com/pybind/pybind11/issues/1831)
+    if url.startswith('../'):
+        return urllib.parse.urljoin('https://github.com/microsoft/onnxruntime', url)
+    return url
 
 def format_component(submod):
-    return {"component": {"type": "git", "git": {"commitHash": str(submod.head_id), "repositoryUrl": submod.url}}}
+    return {"component": {"type": "git", "git": {"commitHash": str(submod.head_id), "repositoryUrl": absolute_url(submod.url)}}}
```
Comment (skottmckay) [inline tools/python/get_submodules.py]: This formatting order seems to match the original cgmanifest.json and not the updated one in this PR where the 'git' entry is prior to the 'type' entry.

File: cgmanifest.json (hunk 1 above)
Comment (tiagoshibata) [reply cgmanifest.json]: Content was updated and there's more submodules than before. I don't know how the old one was generated, this is the order generated by the `get_submodules.py` script

File: tools/python/get_submodules.py (hunk 2 above)
Comment (tiagoshibata) [reply tools/python/get_submodules.py]: Python dicts are hashed dictionaries, with no implicit ordering. The order in which the json is dumped depends on the parameters given to `json.dump` (see line 79 of the script: `json.dump(registrations, f, indent=4, sort_keys=True)`. sort_keys will sort them alphabetically).

Comment (tiagoshibata) [reply tools/python/get_submodules.py]: If someone can point me to how the old one was generated or let me know if it was manually generated please do :)

**Comment by tiagoshibata** [comment]
Ping @skottmckay . WinML wants to use the newer component governance and tools for legal review, so we'd like to get the manifest right in the first place :)

* How is the manifest currently generated? Are we simply updating it by hand? The ordering generated by get_submodules.py was already different before my change, which leads me to believe that it was manually updated.
* How do we want it to be in the future? Are we maintaining this script or will we keep updating the manifest by hand?

[(See my replies to your comments here)](https://github.com/microsoft/onnxruntime/pull/2777#discussion_r363438436)

**Comment by snnn** [comment]
The file was manually created and maintained.

**Comment by tiagoshibata** [comment]
Hey, is it desirable to maintain this tool and fix these bugs or should I just close the PR and we'll keep doing it manually?

**Comment by snnn** [comment]
If the cost of manually maintaining the json file is acceptable for you, I'd rather abandoning the tool

**Pull request closed without merging**